api/src/main/java/org/openmrs/module/patientqueueing/model/PatientQueue.java

```diff
@@ -19,132 +18,132 @@
 @Entity(name = "patientqueueing.PatientQueue")
 @Table(name = "patient_queue")
 public class PatientQueue extends BaseOpenmrsData implements Serializable {
-
-    @Id
-    @GeneratedValue
-    @Column(name = "patient_queue_id")
-    private Integer patientQueueId;
-
-    @ManyToOne
-    @JoinColumn(name = "patient_id")
-    private Patient patient;
```

Comment:
It would be great to add these formatting changes from a pull request of their own